extras/rebalance/directory-rebalance.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/python3
+import argparse
+import os
+import errno
+import time
+import sys
+import datetime
+import hashlib
+import logging
+
+def size_fmt(num, suffix='B'):
+    for unit in ['','Ki','Mi','Gi','Ti','Pi','Ei','Zi']:
+        if abs(num) < 1024.0:
+            return "{:3.1f} {}{}".format(num, unit, suffix)
```

Comment:
`3.1f` format only allows a single digit in the integer part. Was this the intention ? I think it should be `5.1f` at least, or even `6.2f`.

```suggestion
            return "{:6.2f} {}{}".format(num, unit, suffix)
```

Reply:
It prints 3 digits in the integer part and 1 digit in the fraction part. Maybe 4.1 is better?

Reply:
Not really. `3.1` means 3 spaces allocated for the whole number, with one of them reserved for the fractional part.

```python
>>> print("'{:3.1f}'".format(4.56))
'4.6'
>>> print("'{:5.1f}'".format(4.56))
'  4.6'
```

So if you want to keep the numbers always aligned and not use more space when they grow, you should use 5.1 or 6.1 at least.